lib/measures/tbd/measure.rb

```diff
@@ -46,26 +51,61 @@ def exitTBD(runner, out = false, io = nil, surfaces = nil)
 
   io = {} unless io
 
-  seed_file = runner.workflow.seedFile
-  seed_file = seed_file.get.to_s unless seed_file.empty?
-  description = "Thermal Bridging and Derating"
-  description += " - '#{seed_file}'" unless seed_file.empty?
-  io[:description] = description unless io.has_key?(:description)
+  seed = runner.workflow.seedFile
+  seed = seed.get.to_s unless seed.empty?
+  seed = File.basename(seed)
+  descr = "Thermal Bridging and Derating"
+  descr += " - '#{seed}'" unless seed.empty?
+  io[:description] = descr unless io.has_key?(:description)
 
   unless io.has_key?(:schema)
-    io[:schema] = "https://github.com/rd2/tbd/blob/master/tbd.schema.json"
+    io[:schema] = "https://github.com/rd2/tbd/blob/ua/tbd.schema.json"
   end
 
   tbd_log = { date: Time.now, status: status }
+  version = model.getVersion.versionIdentifier
+
+  ua_md_en = nil
+  ua_md_fr = nil
+  ua = nil
+  if surfaces && gen_ua
+    ua = ua_summary(surfaces, tbd_log[:date], version, descr, seed, ref)
+  end
+
+  unless TBD.fatal? || ua.nil? || ua.empty?
+    if ua.has_key?(:en)
+      if ua[:en].has_key?(:b1) || ua[:en].has_key?(:b2)
+        runner.registerInfo("-")
+        runner.registerInfo(ua[:model])
+        tbd_log[:ua] = {}
+        ud_md_en = ua_md(ua, :en)
+        ud_md_fr = ua_md(ua, :fr)
```

Comment:
MD-formatted report content (French & English) is retrieved here , yet only written out to '/files' after the directory sorting is done.